opencxl/cxl/component/virtual_switch/routers.py

```diff
@@ -412,17 +418,22 @@ async def _process_host_to_target_packets(self):
             if target_fld_name not in usp_component.get_cache_route_table_options:
                 logger.warning(self._create_message("Received unroutable CXL.cache packet"))
                 continue
-            target_port = usp_component.get_cache_route_table_options[target_fld_name][
+            target_port: int = usp_component.get_cache_route_table_options[target_fld_name][
                 "port_number"
             ]
-
+            if target_port is None:
+                logger.warning(self._create_message("Received unroutable CXL.cache packet"))
+                continue
```

Comment:
Add more prints here about the packet contents to facilitate debugging. Theoretically, it should never get here.